modules/concept-docs/pages/compression.adoc

```diff
@@ -1,10 +1,11 @@
 = Compression
+:description: In response to increasing volumes of data being sent over the wire, Couchbase Data Platform now provides data compression between the SDK and Couchbase Server.
```

Comment:
Could do with condensing - maybe something along the lines of:
"Data compression to reduce traffic costs from app to Server."